core/src/main/java/org/mskcc/cbio/portal/util/GlobalProperties.java

```diff
@@ -858,28 +858,6 @@ public static String getSessionServicePassword()
         return sessionServicePassword;
     }
 
-    public static String getOncoKBPublicApiUrl()
```

Comment:
We no longer provide oncokb api url to frontend. The website will disable the cross domain.